app/workers/streams/twitch_webhook_registration_worker.rb

```diff
@@ -1,6 +1,7 @@
 module Streams
-  class TwitchWebhookRegistrationJob < ApplicationJob
-    queue_as :twitch_webhook_registration
+  class TwitchWebhookRegistrationWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :default, retry: 10
```

Comment:
@mstruve How do you feel about queue and number of retries here? I'm happy to change either of them whichever way you prefer.

Reply:
I would ask @mstruve or @benhalpern. To be honest I'm not sure the Twitch feature is used that much so I would keep it in the default queue, even low priority maybe. We can always change it

Reply:
Can you put it on `low_priority`? default and low_priority have the same importance but I find people get confused sometimes with the default queue, whereas the low_priority is pretty explicit about its priority.

Reply:
Yup, done.